superheroes.yaml

```diff
@@ -0,0 +1,510 @@
+        else:
+        - abort: failed to capture pod id for Kafka broker
+    - sh: ${{PODMAN}} logs -f ${{HOST.KAFKA_POD_ID}}
+      silent: true # prevent logs to be printed in the terminal
+      watch:
+      - regex: Transition from STARTING to STARTED
+        then:
+        - ctrlC
+      timer:
+        1m: # max wait time
+        - abort: Kafka broker has not been ready in 1 min
+    - signal: KAFKA_READY
+
+  start-fights-rest:
+  - js: ${{FIGHTS_ENABLED}}
+    then:
+    - wait-for: FIGHTS_DB_READY
+    - wait-for: KAFKA_READY
+    - wait-for: OTEL_READY
+    - log: starting Fights rest service...
+    - sh: >
+        ${{PODMAN}} run -d
+        --name rest-fights 
+        -p ${{FIGHTS_REST_PORT}}:8082
+        --memory 1G
+        --cpus 1
+        -e QUARKUS_MONGODB_HOSTS=${{FIGHTS_DB_HOSTNAME}}:${{FIGHTS_DB_PORT}}
+        -e QUARKUS_LIQUIBASE_MONGODB_MIGRATE_AT_START="false"
+        -e QUARKUS_MONGODB_CREDENTIALS_USERNAME=superfight 
+        -e QUARKUS_MONGODB_CREDENTIALS_PASSWORD=superfight
+        -e QUARKUS_STORK_HERO_SERVICE_SERVICE_DISCOVERY_ADDRESS_LIST=${{HEROES_REST_HOSTNAME}}:${{HEROES_REST_PORT}}
+        -e QUARKUS_STORK_VILLAIN_SERVICE_SERVICE_DISCOVERY_ADDRESS_LIST=${{VILLAINS_REST_HOSTNAME}}:${{VILLAINS_REST_PORT}}
+        -e MP_MESSAGING_CONNECTOR_SMALLRYE_KAFKA_APICURIO_REGISTRY_URL="http://${{APICURIO_HOSTNAME}}:${{APICURIO_PORT}}/apis/registry/v2"
+        -e KAFKA_BOOTSTRAP_SERVERS=PLAINTEXT://${{KAFKA_HOSTNAME}}:${{KAFKA_PORT}}
+        -e QUARKUS_OTEL_EXPORTER_OTLP_TRACES_ENDPOINT="${{OTEL_PROTOCOL}}://${{OTEL_HOSTNAME}}:${{OTEL_RECEIVER_PORT}}"
+        ${{= getNetwork( '${{SUT_SERVER}}' ) }}
+        ${{FIGHTS_REST_IMAGE:'quay.io/quarkus-super-heroes/rest-fights:native-latest'}}
+      then:
+      - regex: (?<HOST.FIGHTS_REST_POD_ID>[a-f0-9]{64})
+        else:
+        - abort: failed to capture pod id for Fights rest service
+    - sh: ${{PODMAN}} logs -f ${{HOST.FIGHTS_REST_POD_ID}}
+      silent: true # prevent logs to be printed in the terminal
+      watch:
+      - regex: started in.*Listening on
+        then:
+        - ctrlC
+      timer:
+        1m: # max wait time
+        - abort: Fights rest service has not been ready in 1 min
+    - signal: FIGHTS_REST_READY
+
+  start-fights-ui:
+  - js: ${{UI_ENABLED}}
+    then:
+    - wait-for: FIGHTS_REST_READY
+    - log: starting Fights ui...
+    - sh: >
+        ${{PODMAN}} run -d
+        --name ui-super-heroes
+        -p ${{FIGHTS_UI_PORT}}:8080
+        -e API_BASE_URL=http://${{FIGHTS_REST_HOSTNAME}}:${{FIGHTS_REST_PORT}}
+        ${{= getNetwork( '${{SUT_SERVER}}' ) }}
+        ${{FIGHTS_UI_IMAGE:'quay.io/quarkus-super-heroes/ui-super-heroes:native-latest'}}
+
+      then:
+      - regex: (?<HOST.FIGHTS_REST_UI_ID>[a-f0-9]{64})
+        else:
+        - abort: failed to capture pod id for Fights ui
+
+  cleanup-superheroes-repo:
+  - sh: rm -Rf ${{SUPERHEROES_FOLDER:/tmp/quarkus-super-heroes}}
```

Comment:
we should ignore exit code for this command, because it does not matter if this cmd returns an error